src/System.Configuration.ConfigurationManager/src/System/Configuration/OverrideModeSetting.cs

```diff
@@ -25,17 +25,11 @@ internal struct OverrideModeSetting
 
         private byte _mode;
 
-        internal static OverrideModeSetting s_sectionDefault;
-        internal static OverrideModeSetting s_locationDefault;
+        // Default for section is ALLOW
+        internal static readonly OverrideModeSetting s_sectionDefault = new OverrideModeSetting { _mode = (byte)OverrideMode.Allow };
 
-        static OverrideModeSetting()
```

Comment:
(Just so I can feel like this is a defensive change instead of an actual change).  Reading the CA2207 rule my initial understanding was that only declared struct ctors fired the struct cctor; but perhaps it's that `OverrideModeSetting setting = default;` does not and `OverrideModeSetting setting = new OverrideModeSetting();` does?  Being, therefore, a very subtle difference between the two zero-init modes of a struct?  (And that definite assignment wouldn't fire it either, but DA doesn't apply in this case).

Reply:
The resulting IL will be the same for both the before and after, with the exception that the former won't have `beforefieldinit` and the latter will have `beforefieldinit`.

e.g.
https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDYB8ACBmAAlgQFcBjGAgZQEYBYAKAG9GC2D8OaA2AhOCABMA9gDsMATy4AGAgDFhwgNyt2nHDz4CR4qRtkAhCAhUN2BVWw29aACgCUlgizPn2C4QQC8BAEQAzRV9TN3YjBG8/YGNgpwBfRgSGRk5ickoqACZGF3N1TX4hMUkZeUVIgKCQtUJrLSLdUvCK6IRYhjigA

The `beforefieldinit` gives the runtime more flexibility in when it invokes the cctor and as such allows it to employ various optimizations that it can't otherwise.